### ISSUE TITLE
权限好象不是最大

### ISSUE THREAD

**Issue by ttsite** [antiwar3/py#15]
在虚拟机2012 2019 2016上测试了一下。驱动文件成功写入。但对于没有附于管理员权限的文件和文件夹还有注册表之类的。在工具上面显示可以删除掉。但实际删除不掉。

**Comment by antiwar3** [comment]
强制删除那个功能没用吗




antiwar3
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "antiwar3/py"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月8日(星期四) 晚上8:42
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[antiwar3/py] 权限好象不是最大 (Issue #15)





 
在虚拟机2012 2019 2016上测试了一下。驱动文件成功写入。但对于没有附于管理员权限的文件和文件夹还有注册表之类的。在工具上面显示可以删除掉。但实际删除不掉。

**Comment by ttsite** [comment]
是的。只能删除正常文件和文件夹。一些稍有点权限要删除的注册表. 文件或者文件夹软件显示删除。但实际上是没有删除掉。

**Comment by ttsite** [comment]
是的。删除不掉。假如正常文件 文件夹。注册表。之类在没有设定权限的时侯也就是任何人都可以删除的时侯可以。但只要稍加点权限的这几项都删除不掉。进程管理中结束和删除也是一样。无法结束进程删除文件。

**Comment by ttsite** [comment]
期待您的更新版本
